dev/dependencies.sh

```diff
@@ -32,13 +32,14 @@ SBT_ENABLED="false"
 REPLACE="false"
 CHECK="false"
 MODULE=""
+PROFILE="aws"
```

Comment:
In default, we won't enable aws profile.

Reply:
> In default, we won't enable aws profile.

Fixed. Thanks